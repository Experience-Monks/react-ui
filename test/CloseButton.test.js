import React from 'react';
import { CloseButton } from '../dist/index.module.js';
import renderer from 'react-test-renderer';

describe('CloseButton', () => {
  const component = renderer.create(<CloseButton />);

  let tree;

  it('should render correctly', () => {
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
