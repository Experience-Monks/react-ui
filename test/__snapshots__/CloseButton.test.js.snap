// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloseButton should render correctly 1`] = `
<button
  className="CloseButton"
>
  <span />
  <span />
</button>
`;
